Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -914,59 +915,64 @@ def main():
     if PORT:
         # port was configured, cast to int
         PORT = int(PORT)
-
-    try:
-        server = initialize_server(SERVER_IP, PORT, SECURE_CONNECTION, UNSECURE)
-    except Exception as e:
-        return_error(str(e))
-        return
-    global conn
-    if NTLM_AUTH:
-        # intialize connection to LDAP server with NTLM authentication
-        # user example: domain\user
-        domain_user = SERVER_IP + '\\' + USERNAME if '\\' not in USERNAME else USERNAME
-        conn = Connection(server, user=domain_user, password=PASSWORD, authentication=NTLM)
-    else:
-        # here username should be the user dn
-        conn = Connection(server, user=USERNAME, password=PASSWORD)
-
-    # bind operation is the “authenticate” operation.
+    last_log_detail_level = None
     try:
-        # open socket and bind to server
-        if not conn.bind():
-            message = "Failed to bind to server. Please validate the credentials configured correctly.\n{}".format(
-                json.dumps(conn.result))
+        try:
+            set_library_log_hide_sensitive_data(True)
+            if is_debug_mode():
+                demisto.info('debug-mode: setting library log detail to EXTENDED')
+                last_log_detail_level = get_library_log_detail_level()
+                set_library_log_detail_level(EXTENDED)
+            server = initialize_server(SERVER_IP, PORT, SECURE_CONNECTION, UNSECURE)
+        except Exception as e:
+            return_error(str(e))
+            return
```

Comment:
is this here for the linter?

Reply:
yes. linter doesn't know that this call sys.exit.